test/e2e-tests.sh

```diff
@@ -32,6 +32,10 @@ done
 add_trap "for deployment in 3scale-kourier-control 3scale-kourier-gateway; do \
   kubectl -n ${KOURIER_NAMESPACE} scale deployment $deployment --replicas=1; done" SIGKILL SIGTERM SIGQUIT
 
+# Changing the bucket count and cycling the controllers will leave around stale
+# lease resources at the old sharding factor, so clean these up.
+kubectl -n ${SYSTEM_NAMESPACE} delete leases --all
```

Comment:
Shouldn't this be KOURIER_NAMESPACE? or even both?

Reply:
yep, good catch